crates/nu-command/src/generators/seq_date.rs

```diff
@@ -187,13 +188,14 @@ pub fn run_seq_dates(
 ) -> Result<Value, ShellError> {
     let today = Local::now().date_naive();
     // if cannot convert , it will return error
-    let mut step_size: i64 = increment.as_i64()?;
+    let increment_span = increment.span();
+    let mut step_size: i64 = i64::from_value(increment)?;
```

Comment:
Is that one of those "might be a duration" things (#14147)? 
Will take a look when I get back to #14150